travel-agency-service/src/test/java/cz/muni/fi/pa165/bluebat/facade/CustomerFacadeImplTest.java

```diff
@@ -29,76 +31,111 @@
-        Long id = 1L;
-        Customer customer = getDefaultCustomer();
-        customer.setId(id);
-        CustomerDTO customerDTO = getDefaultCustomerDTO();
-        customerDTO.setId(id);
+        CustomerDTO result=customerFacade.updateCustomer(customerDTO);
 
-        customerFacade.updateCustomer(customerDTO);
+        verify(customerService, times(1)).updateCustomer(any());
+        Assert.assertEquals(result,customerDTO);
+    }
 
-        verify(customerService, times(1)).updateCustomer(customer);
+    @Test
+    public void updateCustomer_null_IllegalArgumentException() {
+        Assertions.assertThrows(IllegalArgumentException.class, () -> customerFacade.updateCustomer(null));
     }
 
     @Test
     public void deleteCustomer_valid() {
-        Long id = 1L;
-        Customer customer = getDefaultCustomer();
-        customer.setId(id);
-        when(customerService.findCustomerById(id)).thenReturn(customer);
-
-        customerFacade.deleteCustomer(id);
+        when(customerService.findCustomerById(CUSTOMER_ID)).thenReturn(customer);
+        customerFacade.deleteCustomer(CUSTOMER_ID);
 
         verify(customerService, times(1)).deleteCustomer(customer);
     }
 
+    @Test
+    public void deleteCustomer_nullId_IllegalArgumentException() {
+        Assertions.assertThrows(IllegalArgumentException.class, () -> customerFacade.deleteCustomer(null));
+    }
+
+    @Test
+    public void deleteCustomer_negative_IllegalArgumentException() {
+        Assertions.assertThrows(IllegalArgumentException.class, () -> customerFacade.deleteCustomer(-1L));
+    }
+
     @Test
     public void getCustomerById_valid() {
-        Long id = 1L;
-        Customer customer = getDefaultCustomer();
-        customer.setId(id);
-        CustomerDTO customerDTO = getDefaultCustomerDTO();
-        customerDTO.setId(id);
-        when(customerService.findCustomerById(id)).thenReturn(customer);
+        when(customerService.findCustomerById(CUSTOMER_ID)).thenReturn(insertedCustomer);
 
-        CustomerDTO result = customerFacade.getCustomerById(id);
+        CustomerDTO result = customerFacade.getCustomerById(CUSTOMER_ID);
 
-        verify(customerService, times(1)).findCustomerById(id);
-        Assert.assertEquals(result.getId(), customerDTO.getId());
-        Assert.assertEquals(result.getAddress(), customerDTO.getAddress());
-        Assert.assertEquals(result.getName(), customerDTO.getName());
-        Assert.assertEquals(result.getBirthday(), customerDTO.getBirthday());
-        Assert.assertEquals(result.getEmail(), customerDTO.getEmail());
-        Assert.assertEquals(result.getSurname(), customerDTO.getSurname());
-        Assert.assertEquals(result.getPhoneNumber(), customerDTO.getPhoneNumber());
+        verify(customerService, times(1)).findCustomerById(CUSTOMER_ID);
+        Assert.assertEquals(result,customerDTO);
     }
+
+    @Test
+    public void getExcursionById_nullId_IllegalArgumentException() {
```

Comment:
přejmenovat test, protože voláš getCustomerById a ne getExcursionById
```suggestion
    public void getCustomerById_nullId_IllegalArgumentException() {
```